index.html

```diff
@@ -134,27 +134,44 @@ <h2>
         <code><dfn>note_taking</dfn></code> member
       </h2>
       <p>
-        The <code>note_taking</code> member of the <a data-cite=
+        The `note_taking` member of the <a data-cite=
         "appmanifest#web-application-manifest">Web Application Manifest</a> is
-        an <a data-cite="appmanifest#dfn-object">object</a> that contains
-        information related to note-taking. It has the following members:
+        an [=ordered map=] that contains information related to note-taking. It
```

Comment:
Not sure if it's correct to say the member is an infra ordered map (ie post processing) or a JSON object (ie. pre processing). WDYT?

Reply:
I think I would say "object" (pre-processing), since you're describing the JSON data structure, not the internal representation after processing.